lists/gt/gt-rpj.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": {
+    "en": "Registry for Legal Entities (Guatemala)",
+    "local": "Registro de las Personas Jurídicas"
+  },
+  "url": "https://mingob.gob.gt/mingobsites/repeju/",
+  "description": {
+    "en": "Registry for both non-profits (churches, civil society organisations, NGOs, foundations, etc) and 'Branches or Agencies of Foreign Entities'. Without this, NGOs cannot operate in Guatemala. Not to be confused with the Registro Nacional de las Personas (for individuals) or the NIT, which is the tax identification number. The length varies, and consists of a combination of several columns in registries or several numbers in a document. The document could say 'partida no 236, folio(s) 413, de(l) (los) libro(s) 45' which results in the ID 23641345."
+  },
+  "coverage": [
+    "GT"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "charity"
+  ],
+  "sector": [],
+  "code": "GT-RPJ",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "secondary",
+  "access": {
+    "availableOnline": true,
+    "onlineAccessDetails": null,
+    "publicDatabase": "https://mingob.gob.gt/mingobsites/repeju/entidades-inscritas/",
+    "guidanceOnLocatingIds": "The website has a list of several PDF files which can be opened and searched by Ctrl-F. The names of the organisations are in Spanish. Combine the numbers in the first four (sometimes three) columns to find the ID.",
+    "exampleIdentifiers": "38634663461, 17272, 23641345",
+    "languages": ["es"]
+  },
+  "data": {
+    "availability": [
+      "pdf"
+    ],
+    "dataAccessDetails": "PDFs are linked to from https://mingob.gob.gt/mingobsites/repeju/entidades-inscritas/",
+    "features": [
+      "date_of_registration",
+      "registration_number",
+      "activities_sector_code"
+    ],
+    "licenseStatus": "no_license",
+    "licenseDetails": ""
+  },
+  "meta": {
+    "source": "Government websites, legal documents provided by organisation",
```

Comment:
I'm unable to find a record for Plan International in any of the pdfs on the site. Nor can I find any Oxfam, Costa Foundation, CAFOD or ActionAid UK on the current NGO list, despite them being listed as working in Guatemala according to https://ngoexplorer.org/country/GTM

Am I missing something?

Reply:
I can't see anything either. On the submission form to register an entity there are 7 categories, including a foreign entities category. On the data publication page there are PDFs for the other 6 categories, and they were published in 2018. Maybe the registration of foreign entities has only been required recently?

I don't think there's much more we can do here.